source/api/tests/test_api_integration.py

```diff
@@ -0,0 +1,380 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+# 
+# ###############################################################################
+# PURPOSE:
+#   * Integration test for project api endpoints and workflow.
+# 
+# USAGE:
+#   python3 -m venv .venv &&
+#   source .venv/bin/activate &&
+#   cd tests &&
```

Comment:
source/api/tests

Reply:
This is to activate the virtual environment